### PR TITLE
Feat/1289/update helm values

### LINKED ISSUE #1289: Use development-minimal.yaml as new default values.yaml
**Is your feature request related to a problem? Please describe.**
Having a default values.yaml that noone uses is confusing, especially with having both a development and a development-minimal configuration.

**Describe the solution you'd like**
Use the development-minimal as the default values.yaml, with the option to use development.yaml to deploy a full version of the stack

**Additional context**
Discussed with @JeremyTheocharis over Slack

### PR DESCRIPTION
**Pull request by DaruZero**
# Description

This PR updates the `values.yaml` file with the content of the `development-minimal.yaml` file, and the `development.yaml` with just the startup script
This way we have a default minimal configuration with the option to customize thel deployment using the new `development.yaml` file.

Fixes #1289 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Deployed on our test server: all pods running without errors

**Test Configuration**:
* Kubernetes Version: 
* Helm Version:
* Hardware:

# Checklist:

- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding changes to the changelog (if needed).
- [ ] I have made corresponding notices to the upgrade instructions (if needed)

# Changelog changes

# Upgrade instructions

### CODE REVIEW

File: docs/static/deployments/development.yaml [hunk 1]
```diff
@@ -186,6 +186,98 @@ write_files:
...
+        extraFlags:
+          - --maxmemory 4gb
+        resources:
+          limits:
+            memory: 4Gi
+          requests:
+            cpu: 100m
+            memory: 1Gi
+      replica:
+        replicaCount: 3
```
Comment (JeremyTheocharis) [inline docs/static/deployments/development.yaml:237]: lets use the original replicas of the development-minimal  (which is 1)

Comment (JeremyTheocharis) [reply docs/static/deployments/development.yaml:237]: also all limits and requests

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: deployment/united-manufacturing-hub/values.yaml [hunk 2]
```diff
@@ -3144,8 +2433,8 @@ vernemq:
       value: "on"
     - name: DOCKER_VERNEMQ_LISTENER__SSL__DEFAULT__REQUIRE_CERTIFICATE
       value: "on"
-    - name: DOCKER_VERNEMQ_LISTENER__TCP__ALLOWED_PROTOCOL_VERSIONS
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/values.yaml:3147]: this value is actually important for the MQTT simulator.

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

**Comment by JeremyTheocharis** [comment]
I need to rebase this on the v0.9.6-prerelease1 branch. A change of the values.yaml and development.yaml can only be executed at the same time with a new version release as it would break otherwise the existing installation process.

**Comment by Scarjit** [comment]
@JeremyTheocharis Can you re-review this, so i can prepare the 0.9.6-pr1 release ?

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**